### ISSUE TITLE
does not actually seed

### ISSUE THREAD

**Issue by jdell64** [frostme/sails-seed#17]
I may be missing something, but am I supposed to run a command to get it to actually seed?

I'm running:

- sails v0.12.1   
- sails-seed@0.4.1 

    info: Starting app...

    info: Your seeds are ready to grow!

Then

    GET http://localhost:1337/model

gives

    [ ]

Seed file:

```
module.exports.seeds = {
  source: [
    {
      "name": "punchIp",
      "urlFormat": "https://www.packetmail.net/iprep.php/{ip}?apikey={apiKey}",
      "apiKey": "testKey",
      "sourceType": "ip",
    }
  ]

}


```

**Comment by frostme** [comment]
quick question, is your /model actually /source? If not they need to be the same. Otherwise I'l continue investigating

**Comment by jdell64** [comment]
Sorry, Yes it is. Model:


```
module.exports = {

  attributes: {
    name: {
      type: 'string'
    },
    urlFormat: {
      type: 'string'
    },
    apiKey: {
      type: 'string'
    },
    sourceType: {
      type: 'string',
      enum: ['ip', 'domain'],
      defaultsTo: 'ip'
    }

  }, 
 
test: function(options, cb){
return 'test'
}


};
```

**Comment by frostme** [comment]
thanks for pointing this out. Turns out the seed functions were loading sync and your app was just so fast, it zoomed but our seed data loading lol so i changed the data to load async in #18 , and i published the new version 0.4.3
